core_lib_generator/config_collectors/database.py

```diff
@@ -85,9 +85,12 @@ def _generate_db_config(
     migrate: bool = False
 ) -> dict:
     config = _build_url(db_type, db_name, db_username, db_password, db_port, db_host)
+    if db_type:
+        db_type: str = 'core_lib.connection.sql_alchemy_connection_registry.SqlAlchemyConnectionRegistry'
     return {
         'env': config['env'],
         'connection': {
+            'type': db_type,

```

Comment:
is this being used by the generator?

Reply:
The generator creates this line and looks for the 'type' key while making the yaml file